src/app/modules/audience/views/engagement/top-videos/entry-details-overlay/entry-details-overlay.component.ts

```diff
@@ -0,0 +1,98 @@
+import { Component, Input, OnDestroy, OnInit } from '@angular/core';
+import { BaseEntryGetAction, KalturaClient, KalturaEntryType, KalturaMediaEntry, KalturaMultiRequest, KalturaMultiResponse, KalturaRequestOptions, KalturaUser, UserGetAction } from 'kaltura-ngx-client';
+import { Unsubscribable } from 'rxjs';
+import { cancelOnDestroy } from '@kaltura-ng/kaltura-common';
+import { map } from 'rxjs/operators';
+import { analyticsConfig } from 'configuration/analytics-config';
+import { AreaBlockerMessage } from '@kaltura-ng/kaltura-ui';
+
+export interface EntryDetailsOverlayData {
+  name: string;
+  type: KalturaEntryType;
+  creator: string;
+  creationDate: Date;
+  duration: number;
+  thumbnailUrl: string;
+}
+
+@Component({
+  selector: 'app-entry-details-overlay',
+  templateUrl: './entry-details-overlay.component.html',
+  styleUrls: ['./entry-details-overlay.component.scss'],
+})
+export class EntryDetailsOverlayComponent implements OnInit, OnDestroy {
+  @Input() entryId: string;
+  
+  private _requestSubscription: Unsubscribable;
+  private _partnerId = analyticsConfig.pid;
+  private _apiUrl = analyticsConfig.kalturaServer.uri.startsWith('http')
+    ? analyticsConfig.kalturaServer.uri
+    : `${location.protocol}//${analyticsConfig.kalturaServer.uri}`;
+  
+  public _data: EntryDetailsOverlayData;
+  public _loading = false;
+  public _errorMessage: AreaBlockerMessage;
+  
+  constructor(private _kalturaClient: KalturaClient) {
+  
+  }
+  
+  ngOnInit(): void {
+    if (!this.entryId) {
+      return;
+    }
+  
+    this._loading = true;
+    
+    if (this._requestSubscription) {
+      this._requestSubscription.unsubscribe();
+      this._requestSubscription = null;
+    }
+    
+    const request = new KalturaMultiRequest(
+      new BaseEntryGetAction({ entryId: this.entryId }),
```

Comment:
@diamond-darrell - I think we can add a DetachedResponseProfile here to get a smaller payload.